_posts/2023-09-18-the-definitive-guide-2-cn.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title:  "Chinese edition of Trino: The Definitive Guide"
+author: Manfred Moser, Martin Traverso, Matt Fuller
+image: /assets/ttdg2-cn-cover.png
+excerpt_separator: <!--more-->
+show_pagenav: false
+---
+
+Trino, Trino, Trino everywhere. Just looking at our website stats and the users
+in our community chat, we know that Trino is going places. We also know that one
+of these places with a large user community is China. And now we have good
+news for you.
+
+<!--more-->
+
+Today, we are happy to announce that a translation of the book [Trino: The
+Definitive Guide]({{ site.url }}/trino-the-definitive-guide.html) to Chinese is
```

Comment:
Should we specify Mandarin? "Chinese" isn't one language and could be Min/Cantonese/Shanghainese/etc.

Reply:
Well ... in the written way its "Simplified Chinese" and its kinda implied .. I think we are more likely to mess up and offend .. the current form is simpler.

Reply:
Agreed, based on my extensive 20s Google search, it's better to be broad rather than specific to one form of Chinese.